bin/improver-nbhood

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
Is this what you want, rather than calling this improver-nbhood.py and running it by
`python improver-nbhood.py`?